velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -80,6 +80,14 @@ std::tm getDateTime(Timestamp timestamp, const date::time_zone* timeZone) {
   return dateTime;
 }
 
+FOLLY_ALWAYS_INLINE
+std::tm getDateTm(Date date) {
```

Comment:
consistency: let's rename this function to getDateTime to match the other function that takes Timestamp.